libs/design/src/molecules/modal/modal-content/modal-content.component.scss

```diff
@@ -0,0 +1,9 @@
+@import '../../../scss/daff-util';
+
+:host {
+	display: block;
+	margin: 0 -24px;
+	max-height: 60vh;
+	overflow: scroll;
```

Comment:
This should be `overflow: auto` so that the scrollbar doesn't show up unnecessarily.

Reply:
updated